src/foremast/templates/configs/configs.json.j2

```diff
@@ -75,7 +75,8 @@
             "enabled": true,
             "index_suffix": "index.html",
             "error_document": " "
-        }
+        },
+        "content_encodings": []
```

Comment:
Can we change this key (and the references) from `content_encodings` to `content_metadata` to reflect the field in AWS; again reducing confusion.

Reply:
yeah, good idea